saleor/graphql/giftcard/schema.py

```diff
@@ -49,6 +50,11 @@ class GiftCardQueries(graphene.ObjectType):
         filter=GiftCardFilterInput(
             description=("Filtering options for gift cards." + ADDED_IN_31)
         ),
+        search=graphene.String(
+            description="Look up gift cards by multiple fields."
```

Comment:
Let's make it more explicit - "Search gift cards by email and name of user who created or used the gift card" - maybe something like that.